MapboxNavigation/NavigationMapView.swift

```diff
@@ -39,6 +39,11 @@ open class NavigationMapView: MGLMapView {
      */
     public static let longManeuverDistance: CLLocationDistance = 1000.0
     
+    /**
+     Minimum distnace the user can tap for a selection to be valid when selecting an alternate route.
+     */
+    public var minimumTapDistanceToSelectAltRoute: CGFloat = 50
```

Comment:
maximum, not minimum. If the tap distance is greater than this, we do not select the route.